page/docs/guide/advancing-time.md

```diff
@@ -7,7 +7,7 @@ Block timestamp can be manipulated by setting the exact time or setting the time
 
 ## Set time
 
-Sets the exact time of the next generated block. All subsequent blocks will keep the set offset.
+Sets the exact time and generate block. All subsequent blocks will keep the set offset.
```

Comment:
```suggestion
Sets the exact time and generates a new block. All subsequent blocks will keep the set offset.
```

Reply:
fixed in bedd89c410df8d46c1a98138aa4cdbb9b53f69eb